# Message format for the required RPMs

int16 req_rpm_fl  # Front-left wheel RPM
int16 req_rpm_fr  # Front-right wheel RPM
int16 req_rpm_bl  # Back-left wheel RPM
int16 req_rpm_br  # Back-right wheel RPM
